Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -113,7 +113,7 @@
   description: uplink-mini-bomb-desc
   productEntity: SyndieMiniBomb
   cost:
-    Telecrystal: 6
+    Telecrystal: 4
```

```suggestion
    Telecrystal: 6
```

Put this back up to 6 and then adjust the minibomb timer to be 3 or 4 seconds. I don't know why minibomb is at 10 seconds, it's way too high for what it is.

---

I'm iffy on this before we have any good explosives for spacing or general-on integrity damage (c4 doesn't count, it's way too targeted with it's damage to achieve it), and the timer being 3-4 seconds allows people to pull nonsense where the minibomb blows before anyone can react, as the explosion is basically a 3x3 and can gib if you stand in close proximity, people can just be taken out of the round permanently with little-to-no way of doing anything about it because they - for example - don't wear headphones and can't hear the sound reliably on the first 3 beeps.

---

The minibomb is meant to be an alphastrike. The macrobomb is suppose to be 10+ seconds to allow for disarming.